phpunit.xml.dist

```diff
@@ -1,34 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<phpunit backupGlobals="false"
-         backupStaticAttributes="false"
-         beStrictAboutTestsThatDoNotTestAnything="false"
-         bootstrap="tests/bootstrap.php"
-         colors="true"
-         convertErrorsToExceptions="true"
-         convertNoticesToExceptions="true"
-         convertWarningsToExceptions="true"
-         processIsolation="false"
-         stopOnError="false"
-         stopOnFailure="false"
-         verbose="true"
->
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" backupGlobals="false" backupStaticAttributes="false" beStrictAboutTestsThatDoNotTestAnything="false" bootstrap="tests/bootstrap.php" colors="true" convertErrorsToExceptions="true" convertNoticesToExceptions="true" convertWarningsToExceptions="true" processIsolation="false" stopOnError="false" stopOnFailure="false" verbose="true" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd">
+    <coverage processUncoveredFiles="true">
+        <include>
+            <directory suffix=".php">./src</directory>
+        </include>
+        <exclude>
+            <directory suffix=".blade.php">./src/</directory>
+        </exclude>
+    </coverage>
     <testsuites>
         <testsuite name="Laravel Test Suite">
             <directory suffix="Test.php">./tests</directory>
         </testsuite>
     </testsuites>
-    <filter>
-        <whitelist processUncoveredFilesFromWhitelist="true">
-            <directory suffix=".php">./src</directory>
-            <exclude>
-                <directory suffix=".blade.php">./src/</directory>
-            </exclude>
-        </whitelist>
-    </filter>
     <php>
         <!--
-        <env name="REDIS_HOST" value="127.0.0.1" />
-        <env name="REDIS_PORT" value="6379" />
-        -->
+            <env name="REDIS_HOST" value="127.0.0.1" />
+            <env name="REDIS_PORT" value="6379" />
+            -->
```

Comment:
why we need this change?

Reply:
This was recommended by PHPunit 9.5:

```
PHPUnit 9.5.4 by Sebastian Bergmann and contributors.

Runtime:       PHP 7.4.14
Configuration: C:\Webroot\laravel_pivot_softdeletes\laravel-pivot-softdeletes\phpunit.xml.dist
Warning:       Your XML configuration validates against a deprecated schema.
Suggestion:    Migrate your XML configuration using "--migrate-configuration"!
```

However this does give an issue with running unit tests with code coverage with PHPUnit 8.5:
```
PHPUnit 8.5.15 by Sebastian Bergmann and contributors.

Runtime:       PHP 7.4.14 with Xdebug 3.0.2
Configuration: C:\Webroot\laravel_pivot_softdeletes\laravel-pivot-softdeletes\phpunit.xml.dist

  Warning - The configuration file did not pass validation!
  The following problems have been detected:

  Line 3:
  - Element 'coverage': This element is not expected.

  Test results may not be as expected.

Error:         Incorrect whitelist config, no code coverage will be generated.
```

So we either revert this change or remove PHPUnit 8.4 from require-dev in composer.json. What has your preference?